comments-by-guideline-and-success-criterion.md

```diff
@@ -1063,7 +1063,11 @@ With this substitution, it would read:
 
 **Guideline 4.1 Compatible:** Maximize compatibility with current and future <INS>**[[assistive technologies](#dfn-assistive-technologies) and accessibility features of software]**</INS>.
 
-##### parsing
+##### parsing21
+
+\[Text for 2.1.1 Parsing goes here.\]
```

Comment:
The level 5 headings seem right.

Obviously this isn't ready to approve for merging yet because it says "text ... goes here".

Somehow this will need to deal with the fact that WCAG 2.0 and WCAG 2.1 have the same normative text for 4.1.1, but different notes.

Reply:
> Obviously this isn't ready to approve for merging yet because it says "text ... goes here".
> 
> Somehow this will need to deal with the fact that WCAG 2.0 and WCAG 2.1 have the same normative text for 4.1.1, but different notes.

Sure, just putting something together so that then editors can pick it up from here.

Plan to discuss with Mary Jo on Monday and potentially at the Editor's meeting on Tuesday

Reply:
Daniel moved around the guidance so this part has been fixed.